alma/alma.go

```diff
@@ -0,0 +1,161 @@
+package alma
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/aquasecurity/vuln-list-update/utils"
+	"github.com/cheggaaa/pb/v3"
+	"golang.org/x/xerrors"
+)
+
+const (
+	almaLinuxDir = "alma"
+	urlFormat    = "https://errata.almalinux.org/%s/errata.json"
+	retry        = 3
+)
+
+var (
+	AlmaReleaseVersion = map[string]bool{
+		"8": true,
+	}
```

Comment:
Any reasons to have a map here?

Reply:
I used to use map because it doesn't allow duplication, but I changed it to slice because there is basically no duplication here.